backend/core/src/seeds/listings/listing-detroit-treymore.ts

```diff
@@ -130,11 +129,6 @@ export class ListingTreymoreSeed extends ListingDefaultSeed {
       }
     )
     await this.unitsRepository.save(unitsToBeCreated)
-    const applicationMethod: ApplicationMethod = await this.applicationMethodRepository.save({
```

Comment:
Are you planning on removing these application methods for all the Detroit seeds?

Reply:
I can remove them if we feel the website links are incorrect.  If we feel they are correct for the seed locations, we can just leave them and know we have the correct logic in place (via the Treymore example).  I can even reinstate it for the Treymore if we think that one is correct.

Reply:
Well it's just using the management company's website; which feels slightly misleading to me if the text says "Apply Online" but when you click through to the management website it tells you to call or pick up an application.
Also for some of the links I'm seeing `http://localhost:3000/listing/[id]/associated-management.rentlinx.com/listings`